ogre2/src/Ogre2RenderEngine.cc

```diff
@@ -483,7 +483,7 @@ bool Ogre2RenderEngine::InitImpl()
   {
     gzerr << "Failed to initialize render-engine" << std::endl;
     gzerr << "Please see the troubleshooting page for possible fixes: "
-          << "https://gazebosim.org/docs/fortress/troubleshooting"
+          << "https://gazebosim.org/docs/ionic/troubleshooting"
```

Comment:
We can use `latest` so we don't have to keep updating this:

```suggestion
          << "https://gazebosim.org/docs/latest/troubleshooting"
```

Reply:
yep updated. [82442de](https://github.com/gazebosim/gz-rendering/pull/1030/commits/82442decc5ea8c1d02ba3457ae01f4aefa8a1e66)